usr/src/cmd/rpcsvc/rpc.rusersd.c

```diff
@@ -216,7 +216,7 @@ getutmpx_3(all, version, limit)
 			/*
 			 * need to free this; done after svc_sendreply.
 			 */
-			*q = (struct utmpidle *)
+			*q = (struct utmpidle *)(uintptr_t)
```

Comment:
You never have to cast the return of `malloc` `calloc` etc, they return `void *`.
Usually what this actually means is that someone didn't `#include <stddlib.h>`.  Casting through a longer type isn't going to help if the return is an implicit `int` as here.

Reply:
Yeah already found this somewhat crazy that its needed didn't think about the obvious, fixed now